backend/migrate_performance_indexes.py

```
@@ -0,0 +1,215 @@
+    # Check if we're in Docker environment
+    if os.path.exists('/database'):
+        return os.path.join('/database', 'tools.db')
+    else:
+        # Local development path
+        return os.path.abspath(os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'database',
+            'tools.db'
+        ))
+
+def create_indexes():
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)